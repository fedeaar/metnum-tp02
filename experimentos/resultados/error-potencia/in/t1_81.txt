-3.093989090803864315e+02 -1.487570374119312699e+00 -5.391408274676849466e+00 4.012557783443139670e+01 6.663260105152253310e+00 2.648760314860380660e+01 -7.655581830571516377e+01 -3.963334400145074454e+01 -1.535578449431217862e+01 2.896130682821323887e+01 1.015405002337963793e+02 -3.756903384635011278e+00 -2.022533878537878493e+01 -4.280423351876203597e+00 2.309698707383061933e+00 7.009764795090679002e+01 1.078475328257971100e+01 1.695051925432900575e+01 1.101150985165401011e+02 3.534027585216938050e+00
-1.487570374119314476e+00 3.558141150021066892e+02 -8.557903556938759593e+00 3.013589700048373121e+00 3.230518059807492293e-01 3.175898161568053268e+00 -5.286319760714119553e+01 -3.055738691285360531e+01 -2.270580904020080837e+01 -3.901943853140538820e+00 1.522534475476745008e+01 -2.951548533894052362e+00 -1.788411642719330885e+01 -3.672450993808705100e+00 -5.123852568335464319e+00 1.853673636618194509e+00 1.054737444875773811e+00 -1.183577322162554424e+01 1.383180027983425298e+01 -2.259898535844189205e-01
-5.391408274676853019e+00 -8.557903556938756040e+00 3.941127412819078586e+02 5.294879625363142495e+00 3.967869406184978964e-01 6.722681417145840221e+00 -1.382492311944254197e+02 -8.024857678199722955e+01 -6.064838621691183818e+01 -1.270768778950906963e+01 3.406873176839244621e+01 -7.756393902009175001e+00 -4.718138567768966141e+01 -9.679359480499851287e+00 -1.410372982600386926e+01 -2.769584840132850723e-02 2.088868887827454390e+00 -3.341875974378449854e+01 2.965820880710463925e+01 -8.702727288862557309e-01
4.012557783443139670e+01 3.013589700048373121e+00 5.294879625363138942e+00 -5.771586497743736572e+02 1.276215633703587926e+01 4.716996185883596837e+01 -5.408269344939466805e+01 -2.199292313693030820e+01 1.193881035736010077e+01 6.546014778798061684e+01 1.750829035443909163e+02 -1.981244440455290690e+00 -6.910878008358838365e+00 -1.674384937260307238e+00 1.428203523766782901e+01 1.371668678618956392e+02 1.965464106801345423e+01 5.629762974781788643e+01 1.949124124921660552e+02 7.516838814866575902e+00
6.663260105152253310e+00 3.230518059807494513e-01 3.967869406184967862e-01 1.276215633703587926e+01 -5.698347729157757158e+02 8.117212454490276841e+00 -1.214698706876036560e+01 -5.462418311296758588e+00 6.979289756025794933e-01 1.078520395330718173e+01 3.032736295399609361e+01 -5.035543387509420210e-01 -2.194354851023600173e+00 -4.935383013746731384e-01 2.106572319923355252e+00 2.317833810087728708e+01 3.366758245452317500e+00 8.786276413719493306e+00 3.358226304317228283e+01 1.251287296313420061e+00
2.648760314860381371e+01 3.175898161568052380e+00 6.722681417145840221e+00 4.716996185883596837e+01 8.117212454490276841e+00 -8.007630953690635351e+02 -1.452275288312124601e+01 -2.408579338233712175e+00 1.647423649890510688e+01 4.378078262846231183e+01 1.071932476361943856e+02 -1.402564580547374373e-01 2.439862450251204962e+00 3.361719760143886404e-01 1.120117489059423477e+01 8.786799872603535277e+01 1.228601728649106839e+01 4.092587639191425808e+01 1.205371911040632540e+02 4.941664216301740353e+00
-7.655581830571514956e+01 -5.286319760714119553e+01 -1.382492311944254197e+02 -5.408269344939467516e+01 -1.214698706876036383e+01 -1.452275288312124069e+01 1.632936429836868513e+02 -4.388471539241768369e+02 -3.639762763464647719e+02 -1.474994002552448649e+02 -1.227145310980934934e+00 -4.258006045599424283e+01 -2.648270069689774004e+02 -5.403947414278083272e+01 -9.766060219899745221e+01 -1.553591354992350944e+02 -1.016756130492447596e+01 -2.568151367931914706e+02 -4.915050620698430350e+01 -1.353404964061033589e+01
-3.963334400145073744e+01 -3.055738691285360176e+01 -8.024857678199722955e+01 -2.199292313693031176e+01 -5.462418311296755924e+00 -2.408579338233717504e+00 -4.388471539241767800e+02 5.152533500216185303e+02 -2.115333157029410813e+02 -7.789185844489779242e+01 2.189799527020414516e+01 -2.518273997854347712e+01 -1.559252109357925349e+02 -3.185156181097463701e+01 -5.532665121284745879e+01 -7.323029561595204484e+01 -3.413303308014143944e+00 -1.430700032402709496e+02 -3.595358945756325308e+00 -6.951984467122782618e+00
-1.535578449431218395e+01 -2.270580904020080482e+01 -6.064838621691184528e+01 1.193881035736011142e+01 6.979289756025777169e-01 1.647423649890508557e+01 -3.639762763464648287e+02 -2.115333157029410813e+02 2.743468597840900429e+02 -3.539321166865114776e+01 8.524603954959040664e+01 -2.044964078353386938e+01 -1.245345476893119354e+02 -2.554146460465113222e+01 -3.767617595462792224e+01 -3.845552405548328778e+00 4.981413855213864927e+00 -8.989121504518979577e+01 7.304141696287247498e+01 -2.507304051431214376e+00
2.896130682821323887e+01 -3.901943853140539709e+00 -1.270768778950907318e+01 6.546014778798061684e+01 1.078520395330717818e+01 4.378078262846231183e+01 -1.474994002552448933e+02 -7.789185844489779242e+01 -3.539321166865114066e+01 -2.196691397119534201e+02 1.692978790807432290e+02 -7.409861507555578264e+00 -4.084811478497665149e+01 -8.590983200623307781e+00 1.226062482762098726e+00 1.127192944956906757e+02 1.771150190336856411e+01 2.136245538480582340e+01 1.823086082526586154e+02 5.529544630676914707e+00
1.015405002337963793e+02 1.522534475476745008e+01 3.406873176839245332e+01 1.750829035443909163e+02 3.032736295399609361e+01 1.071932476361944140e+02 -1.227145310980920723e+00 2.189799527020416647e+01 8.524603954959043506e+01 1.692978790807432574e+02 -6.066226389958164873e+02 2.464046472154332701e+00 2.735396743454239399e+01 4.994506161710950742e+00 4.749884160754280771e+01 3.299574329939895279e+02 4.532890625004003482e+01 1.665634106344413112e+02 4.411848471873780682e+02 1.889164525705279729e+01
-3.756903384635012166e+00 -2.951548533894052806e+00 -7.756393902009176777e+00 -1.981244440455289801e+00 -5.035543387509420210e-01 -1.402564580547365491e-01 -4.258006045599424993e+01 -2.518273997854347712e+01 -2.044964078353386583e+01 -7.409861507555578264e+00 2.464046472154330925e+00 7.545588012963406754e+02 -1.510474565462912011e+01 -3.086033119422977755e+00 -5.326646466899290111e+00 -6.817535946380480105e+00 -2.916635652524008915e-01 -1.373613159234389691e+01 3.572079168261740878e-02 -6.580469264799427842e-01
-2.022533878537878849e+01 -1.788411642719330885e+01 -4.718138567768967562e+01 -6.910878008358841917e+00 -2.194354851023599728e+00 2.439862450251219173e+00 -2.648270069689774004e+02 -1.559252109357925349e+02 -1.245345476893119638e+02 -4.084811478497666570e+01 2.735396743454234070e+01 -1.510474565462911833e+01 5.679145555105999392e+02 -1.903653765486187055e+01 -3.165675244366054741e+01 -3.220060002149991618e+01 -4.131686503762453944e-01 -8.027325566517542654e+01 1.385389086051884178e+01 -3.508426612328383598e+00
-4.280423351876202709e+00 -3.672450993808705100e+00 -9.679359480499849511e+00 -1.674384937260304129e+00 -4.935383013746731384e-01 3.361719760143895286e-01 -5.403947414278083272e+01 -3.185156181097463701e+01 -2.554146460465113933e+01 -8.590983200623306004e+00 4.994506161710951631e+00 -3.086033119422976867e+00 -1.903653765486187055e+01 6.791078171380818276e+02 -6.531621069035842275e+00 -7.068720592032327943e+00 -1.526960718707766618e-01 -1.663206033420799557e+01 2.161486098069662187e+00 -7.444393253742185301e-01
2.309698707383063709e+00 -5.123852568335464319e+00 -1.410372982600386926e+01 1.428203523766782901e+01 2.106572319923355252e+00 1.120117489059423477e+01 -9.766060219899745221e+01 -5.532665121284744458e+01 -3.767617595462792224e+01 1.226062482762100503e+00 4.749884160754280771e+01 -5.326646466899290111e+00 -3.165675244366054741e+01 -6.531621069035843163e+00 1.299051842881478649e+02 1.985240691256103673e+01 4.204451148040228681e+00 -1.355805407621193481e+01 4.750571136396788319e+01 5.234609225341406713e-01
7.009764795090676159e+01 1.853673636618196285e+00 -2.769584840132850723e-02 1.371668678618956392e+02 2.317833810087728708e+01 8.786799872603535277e+01 -1.553591354992350944e+02 -7.323029561595210168e+01 -3.845552405548314567e+00 1.127192944956906473e+02 3.299574329939894142e+02 -6.817535946380480105e+00 -3.220060002149990908e+01 -7.068720592032331496e+00 1.985240691256103673e+01 -2.686764243870752011e+02 3.631456933277212329e+01 8.753289857363375859e+01 3.638681907411487941e+02 1.319004424234662665e+01
1.078475328257971455e+01 1.054737444875774255e+00 2.088868887827453502e+00 1.965464106801345068e+01 3.366758245452317500e+00 1.228601728649107017e+01 -1.016756130492448129e+01 -3.413303308014143056e+00 4.981413855213863151e+00 1.771150190336856056e+01 4.532890625004002771e+01 -2.916635652524008915e-01 -4.131686503762489471e-01 -1.526960718707768838e-01 4.204451148040226904e+00 3.631456933277212329e+01 -7.288593140127509287e+02 1.590756920246337813e+01 5.071100439985340813e+01 2.016144188155333250e+00
1.695051925432901641e+01 -1.183577322162554424e+01 -3.341875974378449854e+01 5.629762974781788643e+01 8.786276413719495082e+00 4.092587639191426518e+01 -2.568151367931914706e+02 -1.430700032402709496e+02 -8.989121504518979577e+01 2.136245538480583051e+01 1.665634106344412544e+02 -1.373613159234389691e+01 -8.027325566517541233e+01 -1.663206033420799557e+01 -1.355805407621193659e+01 8.753289857363377280e+01 1.590756920246338346e+01 -1.478797820246759898e+01 1.721325465719886836e+02 3.400030802068064073e+00
1.101150985165401011e+02 1.383180027983425475e+01 2.965820880710463925e+01 1.949124124921660268e+02 3.358226304317228283e+01 1.205371911040632824e+02 -4.915050620698433193e+01 -3.595358945756325308e+00 7.304141696287247498e+01 1.823086082526586154e+02 4.411848471873781250e+02 3.572079168262096149e-02 1.385389086051884178e+01 2.161486098069662187e+00 4.750571136396788319e+01 3.638681907411487373e+02 5.071100439985340103e+01 1.721325465719886836e+02 -3.612066054375333124e+02 2.053286569659190164e+01
3.534027585216938050e+00 -2.259898535844188094e-01 -8.702727288862563970e-01 7.516838814866575014e+00 1.251287296313420061e+00 4.941664216301739465e+00 -1.353404964061033411e+01 -6.951984467122782618e+00 -2.507304051431213487e+00 5.529544630676914707e+00 1.889164525705279729e+01 -6.580469264799427842e-01 -3.508426612328384486e+00 -7.444393253742184191e-01 5.234609225341406713e-01 1.319004424234662665e+01 2.016144188155333250e+00 3.400030802068063629e+00 2.053286569659190164e+01 -3.493295394193841616e+02
