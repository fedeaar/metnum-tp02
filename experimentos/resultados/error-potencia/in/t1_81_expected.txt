-9.960000000000000000e+02
9.010000000000000000e+02
-8.580000000000000000e+02
-8.300000000000000000e+02
7.750000000000000000e+02
7.570000000000000000e+02
-7.340000000000000000e+02
6.830000000000000000e+02
6.610000000000000000e+02
-6.520000000000000000e+02
-5.720000000000000000e+02
-5.160000000000000000e+02
4.350000000000000000e+02
4.170000000000000000e+02
3.590000000000000000e+02
-3.500000000000000000e+02
-3.280000000000000000e+02
-2.640000000000000000e+02
1.370000000000000000e+02
3.000000000000000000e+00
